src/components/liveblog/liveblogByline.tsx

```diff
@@ -45,15 +44,33 @@ const LiveblogBylineStyles = ({ liveblogBackground }: PillarStyles): SerializedS
             opacity: .8;
         }
     }
+
+    section {
+        margin-top: ${basePx(-1)};
+
+        .byline {
+            width: 80%;
+            float: left;
+            display: inline-block;
+        }
+    }
 `;
 
+const commentCount = ({ liveblogBackground }: PillarStyles): SerializedStyles => css`
+    border-left: solid 1px rgba(220,220,218,.4);
+    button {
+        background: ${liveblogBackground};
+    }
+`
+
 interface LiveblogBylineProps {
     byline?: string;
     pillarStyles: PillarStyles;
     publicationDate: string;
     contributors: Contributor[];
     article: Article;
     imageSalt: string;
+    commentable: boolean;
```

Comment:
If this will in future also need the count, could we combine both pieces of information into a single type? Where a value of `Some(<number_of_comments>)` would both display the comment component *and* pass it the number of comments, and a value of `None` wouldn't display the comment component at all.

```suggestion
    commentCount: Option<number>;
```

Reply:
👍 Maybe one for when we get the MAPI data?